browser/ui/views/toolbar/brave_app_menu_browsertest.cc

```diff
@@ -67,6 +70,26 @@ class BraveAppMenuBrowserTest : public InProcessBrowserTest {
 #endif
 };
 
+IN_PROC_BROWSER_TEST_F(BraveAppMenuBrowserTest, AppMenuButtonUpgradeAlertTest) {
+  // Check property for our style.
+  auto* brave_menu_button =
+      static_cast<BraveBrowserAppMenuButton*>(menu_button());
+  EXPECT_TRUE(brave_menu_button->ShouldPaintBorder());
+  EXPECT_TRUE(brave_menu_button->ShouldBlendHighlightColor());
+
+  // Check our highlight color.
+  dark_mode::SetBraveDarkModeType(
+      dark_mode::BraveDarkModeType::BRAVE_DARK_MODE_TYPE_LIGHT);
+  EXPECT_EQ(dark_mode::BraveDarkModeType::BRAVE_DARK_MODE_TYPE_LIGHT,
+            dark_mode::GetActiveBraveDarkModeType());
+  EXPECT_EQ(brave_menu_button->GetHighlightColor(), std::nullopt);
+  brave_menu_button->SetTypeAndSeverity(
+      {AppMenuIconController::IconType::UPGRADE_NOTIFICATION,
+       AppMenuIconController::Severity::LOW});
+  EXPECT_EQ(brave_menu_button->GetHighlightColor(),
+            SkColorSetRGB(0x00, 0x46, 0x07));
```

Comment:
Should we also check other severity levels?

Reply:
good catch. updated to check all levels.